schemas/org.openweathermap.snowplow/weather_stamp/jsonschema/1-0-0

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "org.openweathermap.snowplow",
+    "name" : "weather_stamp",
```

Comment:
I think by definition any entity snapshot (such as the weather) is a snapshot at a specific point in time, therefore the _stamp is superfluous. As an example, if we have an e-comm event describing a shopping basket at the time of the sale, we pass the basket in as `cart`, not `cart_stamp`.